dll/win32/syssetup/wizard.c

```diff
@@ -939,7 +939,7 @@ ComputerPageDlgProc(HWND hwndDlg,
                         return TRUE;
                     }
 
-#if 0
+#if PASSWORDS_MANDATORY
```

Comment:
I don't see where this is defined :thinking: https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=PASSWORDS_MANDATORY

Reply:
I am going to add it shortly.

Reply:
Yes, that's the point. This doesn't enable requiring passwords, it just allows you to enable it using a define.

Reply:
```suggestion
#ifdef PASSWORDS_MANDATORY
```
Then maybe something like that. And `//#define PASSWORDS_MANDATORY` commented definition somewhere.